viewing_party/party.py

```diff
@@ -1,23 +1,415 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """
+    Input: title (string), genre (string), rating (float)
+    Output: If parameters are truthy, return dictionary. If not, return None
+    """
+    movies = {}
+
+    if (title and genre and rating):
+        movies["title"] = title
+        movies["genre"] = genre
+        movies["rating"] = rating
+        return movies

```

Comment:
One note, since this is a single movie not a collection of them the variable should be named `movie`.

Also, I'd generally construct a dictionary literal for something like this instead of making an empty dictionary and modifying it:

```suggestion
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }

```